bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BVMExecutor.java

```diff
@@ -262,7 +230,7 @@ private static BValue populateReturnData(StrandCallback strandCallback, Callable
      *
      * @param programFile to be invoked.
      */
-    private static void invokePackageInitFunctions(ProgramFile programFile) {
+    public static void invokePackageInitFunctions(ProgramFile programFile) {
         for (PackageInfo info : programFile.getPackageInfoEntries()) {
             BValue result = execute(programFile, info.getInitFunctionInfo(), new BValue[0], null, true);
```

Comment:
Seems like we always pass `null` as the 4th argument and `true` as the 5th argument. Shall we refactor them in this PR as well?